src/Mod/Path/App/Area.cpp

```diff
@@ -2615,7 +2615,7 @@ struct ShapeInfo{
                         GeomAPI_ProjectPointOnCurve gpp;
                         gpp.Init( myBestPt,curve);
```

Comment:
please fix whitespace here as well

Reply:
OKay, I have done it. It's my first time to commit a pull request on GitHub, some operations are not quiet familiar which may consume your time.